rest/adminapitest/admin_api_test.go

```diff
@@ -4298,19 +4298,33 @@ func TestDatabaseConfigAuditAPI(t *testing.T) {
 	responseBody = nil
 	require.NoError(t, json.Unmarshal(resp.Body.Bytes(), &responseBody))
 	assert.Equal(t, true, responseBody["enabled"].(bool))
-	assert.False(t, responseBody["events"].(map[string]interface{})[base.AuditIDISGRStatus.String()].(bool), "audit enabled event should be disabled by default")
-	assert.True(t, responseBody["events"].(map[string]interface{})[base.AuditIDPublicUserAuthenticated.String()].(bool), "public user authenticated event should be enabled by default")
+	eventsMap, ok := responseBody["events"].(map[string]interface{})
```

Comment:
is it worth testing/marshalling to the  the concrete type of event?

Reply:
I'm kind of already doing that in two steps, with the `map[string]interface{}` and then `bool` type assertion.

Was there anything more you were thinking?

Reply:
Ugh, I forgot this was `map[string]any` as stored. Disregard this comment.